priv/event_store/migrations/v1.4.0.sql

```diff
@@ -0,0 +1,7 @@
+-- Add annotations field to subscriptions table
+
+ALTER TABLE subscriptions
```

Comment:
I am confused about this file btw, do we even need these files since we use the SQL init module?